src/main.rs

```diff
@@ -1,21 +1,54 @@
 extern crate docopt;
 extern crate xml;
+extern crate toml;
 
-mod library;
-mod parser;
-
+use std::fs::File;
+use std::io::prelude::*;
 use docopt::Docopt;
 use library::*;
 
+mod gobjects;
+mod library;
+mod parser;
+
 static USAGE: &'static str = "
-Usage: spore -d <dir> <lib>
+Usage: gir [-d <girs_dir>] [<gir>]
+
+Options:
+    -d PATH            Directory for girs
 ";
 
 fn main() {
     let args = Docopt::new(USAGE).unwrap()
         .parse().unwrap_or_else(|e| e.exit());
+    //println!("{:?}", args);
+    //println!("Dir: {}", args.get_str("-d"));
+    //println!("File: {}", args.get_str("<gir>"));
+    
+    let toml = read_toml("Gir.toml");
+    // println!("Toml: {:?}", toml);
+    
+    let girs_dir = match args.get_str("-d") {
+        "" => toml.lookup("options.girs_dir")
```

Comment:
I still believe this needs a more specific name. What's a "gir"? `gir = "Gtk-3.0"`? Could be `library`, `namespace`, something with a meaning.